src/test/java/picard/analysis/CollectWgsMetricsTestUtils.java

```diff
@@ -7,39 +7,30 @@
 import java.io.File;
```

Comment:
Is it sufficient to copy one from another file, say TheoreticalSensitivityTest.java? (it's dated 2015 but I couldn't a newer license.) Or is there an automated license update script like GATK?  Also, a couple other tests are also missing licenses (e.g. CollectWgsMetricsTest.java), should I update them as well?